org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/entitytemplates/artifacttemplates/ArtifactTemplateResource.java

```diff
@@ -86,6 +92,30 @@ public void synchronizeReferences() throws IOException {
 		BackendUtils.synchronizeReferences((ArtifactTemplateId) this.id);
 	}
 
+	@POST
+	@Consumes(MediaType.APPLICATION_JSON)
+	public Response copySourceToFilesResource(ArtifactResourcesApiData data) {
```

Comment:
Could you document that `data` is used to state that only a subset of files has to be copied? Use the [@ApiParam](https://github.com/swagger-api/swagger-core/wiki/Annotations-1.5.X#apiparam) value.